packages/charting/index.html

```diff
@@ -17,11 +17,23 @@
         // fontBaseUrl: ''
       };
 
-      var isLocalHost = document.location.href.indexOf('localhost') !== -1 || document.location.href.indexOf('127.0.0.1') !== -1;
+      function getParameterByName(name, url) {
+        if (!url) {
+          url = window.location.href;
```

Comment:
You can use `window.location.search.substr(1)` to get query string

Reply:
Not in this case, see above